CIFAR10/train/lambda_BaiduNet9_lambda_cloud_single_pytorch.json

```
@@ -0,0 +1,24 @@
+{
+    "version": "v1.0",
+    "author": "Chuan Li",
+    "authorEmail": "c@lambdal.com",
+    "framework": "fastai / Pytorch 1.0.0",
+    "codeURL": "https://github.com/lambdal/CIFAR10-Distributed-Training-BaiduNet9P/blob/baea41d37bebb8d6504e7779bb5dff4605953707/train_cifar10_BaiduNet9P.py",
+    "model": "BaiduNet9",
+    "hardware": "Lambda GPU Cloud - gpu.4x instance",
```

Please add the type of GPU as shown below.

```suggestion
    "hardware": "Lambda GPU Cloud - 4x GTX 1080 Ti",
```

---

We've accepted these changes and updated our fork.